pkg/file/scripts/install_exe.ps1

```diff
@@ -0,0 +1,21 @@
+$exeFilePath = "$INSTALLER_PATH"
```

Comment:
Won't the other variables will be replaced by empty string by `os.Expand`?

Reply:
If that's the case, you could return `"$" + placeholder` instead of empty string so that it stays the same (only thing is you can't know if it was `${var}` or just `$var`, so maybe best to always return `${placeholder}`).

Reply:
@mna that's a good catch, I should write better tests for this.

your last comment makes me wonder if we should use a different syntax for "our" variables and `strings.Replace`?

Reply:
damn, yes this didn't work very well with variable declarations in powershell. I will resort to `strings.Replace` for now.

Reply:
yeah makes sense to me.